src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -129,6 +129,20 @@ object DockerBuildPlugin extends AutoPlugin {
     )
 
     ////////////////////////////////////////////////////////////////////////////////////////////////
+    // The following settings affect how the plugin behaves, but don't affect file or image output.
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+
+    val verifyDockerfileIsStrict: SettingKey[Boolean] = Def.settingKey[Boolean](
+      "If true, the `verifyDockerfile` task will raise an error if the current Dockerfile does " +
+        "not match the generated one. Defaults to true."
+    )
+
+    val verifyDockerfileOnBuild: SettingKey[Boolean] = Def.settingKey[Boolean](
+      "If true, `dockerBuild` will depend on `verifyDockerfile`, and will not build if the " +
+        "Dockerfile is not up-to-date. Defaults to false."
```

Comment:
I take that back - it's easy, but not very useful, since the warning gets swallowed in the rest of the build output.

I think forcing this to error is probably a better solution.